oktoberfest/predict/dlomix.py

```diff
@@ -0,0 +1,163 @@
+import contextlib
+import logging
+import multiprocessing as mp
+import os
+import sys
+import warnings
+from pathlib import Path
+from typing import Dict, Union
+
+import numpy as np
+from spectrum_fundamentals.constants import ALPHABET, ANNOTATION
+from spectrum_fundamentals.mod_string import parse_modstrings
+from spectrum_io.file.parquet import write_file
+
+sys.path.append("/cmnfs/home/students/j.schlensok/dlomix/bmpc_shared_scripts/oktoberfest_interface")
+# Suppress unnecessary feature processor info from DLomix
+with open(os.devnull, "w") as devnull, contextlib.redirect_stdout(devnull):  # noqa: E402
+    from oktoberfest_interface import (
+        download_model_from_github,
+        load_keras_model,
+        process_dataset,
+    )
+
+from ..data.spectra import Spectra  # noqa: E402
+
+logger = logging.getLogger(__name__)
+
+ANNOTATIONS = [f"{ion_type}{pos}+{charge}".encode() for ion_type, charge, pos in list(zip(*ANNOTATION))]
```

Comment:
if redefinition is necessary we may want to create this directly in the format required. There is no other reference in any of the oktoberfest packages to this constant (afaik). Plus, this changes depending on what comes from Koina, i.e. we take the annotation output of koina and match it against what is written in `.varnames` in the `Spectra` object. Does `Dlomix` not return this as well? The output of a dlomix prediction should be exactly the same as what comes back from Koina right?

Reply:
See [b75faa5](https://github.com/wilhelm-lab/oktoberfest/pull/250/commits/b75faa5003a089af75d2ab0100ee2d1e502539a2) for new, more dynamic solution